src/animationLoop.js

```diff
@@ -86,7 +86,7 @@ export default function configAnimation(config = {}) {
 
   return function startAnimation(state, animationStep, animationRender) {
     for (let i = 0; i < animRunning.length; i++) {
-      let val = animRunning[i];
+      const val = animRunning[i];
```

Comment:
Probably not as it'll require some pretty sophisticated static analysis.
I like const to mean "truly immutable" in this codebase because at least that's a useful thing to have, as opposed to "this doesn't mean much" from the current formal definition.